core/storage/suggestion/gae_models.py

```diff
@@ -477,8 +477,9 @@ def get_new_suggestions_waiting_for_review(
         current_time_millisecs = utils.get_current_time_in_millisecs()
 
         threshold_datetime = datetime.datetime.utcfromtimestamp(
-            current_time_millisecs / 1000.0) - datetime.timedelta(
-            days=SUGGESTION_REVIEW_WAIT_TIME_NOTIFICATION
+            current_time_millisecs / 1000.0
+            ) - datetime.timedelta(
```

Comment:
Deindent this and the next line by 4 since you are going back to the outer scope.

Reply:
Done